src/geocat/comp/__init__.py

```diff
@@ -9,6 +9,7 @@
 from .fourier_filters import (fourier_band_block, fourier_band_pass,
                               fourier_filter, fourier_high_pass,
                               fourier_low_pass)
+from .gradient import gradient
```

Comment:
If the other functions are intended to be public API, they should be included here, yes?

```suggestion
from .gradient import gradient, arc_lon_wgs84, arc_lat_wgs84, rad_lat_wgs84
```